addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -364,6 +364,9 @@ const Wysiwyg = Widget.extend({
                     && !$target.find('> [data-oe-model]').length
                     && !$target[0].closest('.o_extra_menu_items')
                     && $target[0].isContentEditable) {
+                if (ev.ctrlKey || ev.metaKey) {
```

Comment:
Meta key should only replace ctrl on macos. To match conventions, we ideally want that ctrl+click does _not_ open the link on mac and meta+click does _not_ open the link on non-mac.

Now that I think about it, ctrl+click is not going to be triggered on mac anyway since ctrl+click is the same as a right click (yeah, don't ask), but I don't know if meta+click usually open links on non-mac devices. Could you please check on your end ? If it does open links with meta+click as well then your code exactly matches that behavior and it's fine.

Reply:
I can confirm the behavior on linux for Meta+click: open the link in the _same_ window, then open the "Meta menu". So, is it weird if our behavior is to open the link in a new window ?

Reply:
I'd say not weird enough to warrant a change. Especially since opening in the same page in our case would make no sense IMO.